### ISSUE TITLE
Chrome mobile - strange behavior of input box

### ISSUE THREAD

**Issue by kyvaith** [mathquill/mathquill#643]
Basing on information from issue #65, I should expect MathQuill to not to work on Chrome browser on mobile device, and indeed - after tapping MathQuill input box, device native virtual keyboard shows up, but non of keys pressed with it shows up in input box. What I discovered is after attaching physical keyboard to Android device with USB OTG, and typing at least one letter or digit with it and disconnecting it, virtual keyboard starts to work as excepted. Is it known behavior, or is it something new?

**Comment by laughinghan** [comment]
News to me. Are you willing to help debug this? Can you listen for keydown,
keypress, keyup, and input events on MathQuill's textarea and record the
sequence of them that's fired when typing with the virtual keyboard vs the
physical keyboard? Also, can you record the sequence of them fired on a
normal, non-MathQuill textarea?

Depending on how deep you're willing to dive in on this, this stuff is
solely the responsibility of saneKeyboardEvents.util.js, which is heavily
commented and hopefully helpful.

Han

**Comment by scaljeri** [comment]
I'm experiencing the same behaviour, Chrome on Android shows weird behaviour if you try to type using the native keyboard. Although it doesn't seem to work, you can type whatever you want, but it doesn't show, but if you hit `enter` it suddenly works. 

This behaviour can be reproduced on the mathquill demo site. Is this a known issue yet ? 

I'm also very interested in a temporarily solution ? 

Cheers

**Comment by ajhyndman** [comment]
I've been playing with this a little today.

My impression is: until the `'Enter'` key is pressed, the `typedText` checker is not running.  This is the only way MathQuill is getting text out of the textarea, since key events all return 227 keycodes in Chrome on Android, so only after the first `'Enter'` keypress does the MathQuill input start working.

![image](https://user-images.githubusercontent.com/11449340/33817546-34fbb9c8-de94-11e7-9404-e9ec6df90128.png)

**Comment by ajhyndman** [comment]
It does appear that the only unique property of `'Enter'` is that it fires a `'keypress'` event.

I tested locally, and #694 does fix this issue for me.

As a side-note, that PR doesn't appear to be included in the version of MathQuill running on MathQuill's homepage?  Maybe it's time for an update?

**Comment by ajhyndman** [comment]
I'm closing this issue (and #65) as fixed by #694.

If anyone runs into this again, please feel free to reopen or create a new issue!
